.github/pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+<!--
+
+*Description of changes:*
+
+Describe your changes here, linking to an issue if available.
+
+*Licensing*
+
+To maintain the integrity of the project, we require contributions to be licensed under the Apache License Version 2.0. Please acknowledge this by checking the following check-box. By checking the box, you also acknowledge that you have the authority to license your contribution in this manner. You should also ensure that any new file you add contains "# SPDX-License-Identifier: Apache-2.0" in the header.
+
+ - [ ] I hereby declare this contribution to be licenced under the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2).
```

Comment:
looks pretty good.

Please:
* change the hyper link to `https://apache.org/licenses/LICENSE-2.0`
* *licenced* to **licensed**.

Reply:
done!